Minji/38_깊이우선탐색순회.py

```diff
@@ -0,0 +1,28 @@
+def solution(graph, start):
+  res = []
+  nodes_list = {}
+  visited = []
+  
+  # 인접 리스트 생성
+  for node in graph:
+    if node[0] not in nodes_list:
+      nodes_list[node[0]] = [node[1]]
+    else:
+      nodes_list[node[0]].append(node[1])
```

Comment:
node_list를 defaultdict(list)로 선언하면 해당 로직을 간결하게 할 수 있을 것 같네요
```suggestion
    nodes_list[node[0]].append(node[1])
```